docs/en/jpql-with-kotlin-jdsl/expressions.md

```diff
@@ -247,6 +249,8 @@ ln(path(Book::price))
 
 mod(path(Employee::age), 3)
 
+power(path(Book::verticalLength), 2)
```

Comment:
Is the `Book::verticalLength` property new?
It would be better to use an existing property (e.g. Employee::age)

Reply:
Oh, right, I thought about adding that field, but it's not currently there. 
I know someone added an [age field](https://github.com/line/kotlin-jdsl/pull/648) recently haha I'll replace it with that.